plume/src/main/kotlin/io/github/plume/oss/passes/graph/BaseCPGPass.kt

```diff
@@ -660,6 +711,54 @@ class BaseCPGPass(private val g: BriefUnitGraph) {
         }
     }
 
+    private fun projectFieldAccess(fieldRef: FieldRef, childIdx: Int): NewCallBuilder{
+        val fieldAccessVars = mutableListOf<NewNodeBuilder>()
+        val leftOp = when(fieldRef){
+            is StaticFieldRef ->  fieldRef.fieldRef.declaringClass().type
+            is InstanceFieldRef -> fieldRef.base.type
+            else -> fieldRef.fieldRef.declaringClass().type
+        }
+
+        val fieldAccessBlock = NewCallBuilder()
+                .name(Operators.fieldAccess)
+                .code(fieldRef.toString()) // Review-TODO: Check Code is valid
+                .signature("(${leftOp.toQuotedString()}).(${fieldRef.toString()})") // Review-TODO: Signature of  [(Static/Instance) Field = Signature of Right Tree]
```

Comment:
When I look at an example field access from joern, I see the signature is empty e.g.:
```scala
g.call.filter(_.name == "<operator>.fieldAccess").l 
res2: List[Call] = List(
  Call(
    id -> 1000000032L,
    code -> "java.lang.System.out",
    name -> "<operator>.fieldAccess",
    order -> 2,
    methodFullName -> "<operator>.fieldAccess",
    argumentIndex -> 2,
    signature -> "",
    lineNumber -> Some(value = 9),
    columnNumber -> None,
    methodInstFullName -> None,
    typeFullName -> "java.io.PrintStream",
    depthFirstOrder -> None,
    internalFlags -> None,
    dispatchType -> "STATIC_DISPATCH",
    dynamicTypeHintFullName -> List()
  )
)
```
So you can simply make this a blank string ""